content/pt-br/blog/_posts/2023-02-14-free-katacoda-kubernetes-tutorials-are-shutting-down.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: "Os tutoriais gratuitos do Katacoda Kubernetes estão sendo encerrados"
+date: 2023-02-14
+slug: kubernetes-katacoda-tutorials-stop-from-2023-03-31
+evergreen: true
+---
+
+**Author**: Natali Vlatko, SIG Docs Co-Chair para Kubernetes
```

Comment:
Author can be translated here. I also suggest using something like "co-presidente do SIG Docs do Kubernetes", as it sounds more idiomatic for a native Portuguese reader.